jetstream/pull.go

```diff
@@ -642,7 +652,8 @@ func (p *pullConsumer) fetch(req *pullRequest) (MessageBatch, error) {
 				}
 				userMsg, err := checkMsg(msg)
 				if err != nil {
-					if !errors.Is(err, nats.ErrTimeout) && !errors.Is(err, ErrNoMessages) && !errors.Is(err, ErrMaxBytesExceeded) {
+					if !errors.Is(err, nats.ErrTimeout) && !errors.Is(err, ErrNoMessages) && !errors.Is(err,
+						ErrMaxBytesExceeded) {
```

Comment:
Moving a single parameter to new line looks a bit ugly - either leave everything in single line or split conditions to separate lines.

Reply:
Yes looks like my formatter is doing something crazy